api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -447,8 +446,18 @@ public CohortDefinition getPatientsWithViralLoadLessThan1000Within12Months() {
             + "                OR "
             + "                (o.concept_id=${hivViralLoadQualitative} AND o.value_coded IN (${beyondDetectableLimit},${undetectableViralLoad},${lessThan10Copies},${lessThan20Copies},${lessThan40Copies},${lessThan400Copies},${lessThan839Copies})) "
             + "            )  "
-            + "        AND e.location_id= :location  "
-       
+            + "        AND e.location_id= :location "
+            + "AND (  "
```

Comment:
Were changes on this file required? or mvn clean install will iron those out

Reply:
I guess it changed by `mvn clean install`